client/pkg/logutil/zap.go

```diff
@@ -15,10 +15,10 @@
 package logutil
 
 import (
-	"sort"
-
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"sort"
+	"time"
```

Comment:
```suggestion
	"sort"
	"time"

	"go.uber.org/zap"
	"go.uber.org/zap/zapcore"
```

Reply:
Fixed thanks, make verify told me to change this initially for some reason.